docs/src/usage/std.md

```diff
@@ -67,15 +67,22 @@ This field is used for allowing smarter introspection of how the argument given
 - "read" - This argument is only read from. Currently unused.
 - "write" - This argument is only written to. Used by `unused_variable` to assist in detecting a variable only being written to, even if passed into a function.
 
+#### "must_use"
+This field is used for checking if the return value of a function is used.
+
+- `true` - The default. The return value of this function must be used.
+- `false` - The return value of this function does not need to be used.
+
 Example:
 ```yml
- table.insert:
+table.find:
   args:
     - type: table
-      observes: write # This way, `table.insert(x, 1)` doesn't count as a read to `x`
+      observes: read  # The table is only read from
+      required: true  # The table to index is a required argument
     - type: any
-    - required: false
-      type: any
+      required: true  # The value to find is a required argument
```

Comment:
This is a section for `must_use`, not `required`. This comment isn't needed. Same with the others.

Reply:
See https://github.com/Kampfkarren/selene/pull/583#discussion_r1465686806 for what I was attempting to do.

Reply:
Resolved in: https://github.com/Kampfkarren/selene/pull/583/commits/607ca211d56b7850df50006631eceb5ebc9c87a6